docker/Dockerfile

```
@@ -18,7 +18,7 @@ RUN groupadd --gid $groupid app && \
 
 # install a few essentials and clean apt caches afterwards
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends apt-transport-https build-essential curl git libpq-dev gettext libffi-dev && \
+    apt-get install -y --no-install-recommends apt-transport-https build-essential curl git libpq-dev gettext libffi-dev tcpdump && \
```

```suggestion
    apt-get install -y --no-install-recommends apt-transport-https build-essential curl git libpq-dev gettext libffi-dev && \
    apt-get install -y --no-install-recommends tcpdump && \
```

We don't want to forget we added this, so we should have it on an isolated line and we should also add a FIXME comment about removing it.

Can you also add a FIXME comment for removing the tcpdump thing when we're done?